contrib/endpoints/src/api_manager/request_handler.cc

```diff
@@ -29,6 +29,17 @@ using google::devtools::cloudtrace::v1::Traces;
 namespace google {
 namespace api_manager {
 
+RequestHandler::RequestHandler(ApiManagerImpl* api_manager,
+                               std::shared_ptr<CheckWorkflow> check_workflow,
+                               std::unique_ptr<Request> request_data)
+    : api_manager_(api_manager),
+      check_workflow_(check_workflow),
+      request_data_(std::move(request_data)) {
+  if (api_manager->IsConfigLoadingSucceeded()) {
```

Comment:
This is a real issue.

While config is loading,  if a grpc transcoding request comes,  it needs config_id to find service_config to build trancoder_factory. 

Check() can be delayed by using its callback.   Not sure how easy to delay transcoding factory.
@lizan ?